recipe/meta.yaml

```diff
@@ -31,13 +31,13 @@ requirements:
     - python
     - pip
     - setuptools
-    - llvmlite 0.33.0
+    - llvmlite 0.34.0
     - numpy
     - tbb-devel >=2019.5  # [linux]
 
   run:
     - python
-    - llvmlite >=0.33.0,<0.34
+    - llvmlite >=0.34.0,<0.35
```

Comment:
`numba` version `0.51.0` requires `llvmlite` version `0.34.0` as [noted here]( https://github.com/numba/numba/blob/3a14664a0d7fb1ffdf086e9d02b715e1e2d0e0b3/setup.py#L15-L16 ). Have updated this accordingly. However we currently don't have an `llvmlite` package as noted in issue ( https://github.com/conda-forge/llvmlite-feedstock/issues/42 ). So this is blocked until that's resolved.